app/presenters/user_presenter.rb

```diff
@@ -8,6 +8,7 @@ def initialize(user)
   def as_json(options={})
     return {} unless user
     return {id: user.id, username: '(deleted user)'} if user.deleted?
-    user.as_json_without_presenter({only: [:id, :username]}.reverse_merge(options))
+    attrs = %w(id username moiety moiety_name created_at)
```

Comment:
given my general feelings about moieties, could this maybe be returned in... many fewer places xD

Reply:
I couldn't figure out where we return users – I checked for explicit `to_json` calls. Would I have to check the whole API? :/

Reply:
Oh, I guess maybe our examples are comprehensive enough that I can focus on those?

Reply:
Is this better? I put it in users#index only, but it still shows up three times.